portfolio/src/main/webapp/style.css

```diff
@@ -31,18 +17,41 @@ table.allTable{
     font-family: Tahoma, Geneva, sans-serif;
 }
 
-#socials-row{
+td#about-me-info{
```

Comment:
td#about-me-info can simply be #about-me-info. Same as the rest of the style selectors down below where you combine an element selector and an id or class selector – adding the element selector is superfluous